quinn-proto/src/connection/streams.rs

```diff
@@ -1124,6 +1161,14 @@ impl Recv {
         }
     }
 
+    fn read_chunk(&mut self) -> Result<Option<Bytes>, ReadError> {
+        if let Some(bytes) = self.assembler.read_chunk()? {
+            Ok(Some(bytes))
+        } else {
+            self.read_blocked().map(|()| None)
+        }
```

Comment:
A `match` expression would be more readable and more compact, I think?

Reply:
done